components/analytics/team_analytics/team_analytics.tsx

```diff
@@ -43,24 +60,29 @@ export default class TeamAnalytics extends React.PureComponent {
         locale: PropTypes.string.isRequired,
         stats: PropTypes.object.isRequired,
 
-        actions: PropTypes.shape({
-
-            /*
-             * Function to get teams
-             */
-            getTeams: PropTypes.func.isRequired,
-
-            /*
-             * Function to get users in a team
-             */
-            getProfilesInTeam: PropTypes.func.isRequired,
-        }).isRequired,
+        // actions: PropTypes.shape({
+
+        //     /*
+        //      * Function to get teams
+        //      */
+        //     getTeams: PropTypes.func.isRequired,
+
+        //     /*
+        //      * Function to get users in a team
+        //      */
+        //     getProfilesInTeam: PropTypes.func.isRequired,
+        // }).isRequired,
+        actions: {
+            // getTeams: (a: Team, b: Team) => { data: Team };
+            // getProfilesInTeam: PropTypes.func.isRequired;
+        }
     }
 
-    constructor(props) {
+    constructor(props: Props) {
         super(props);
 
         this.state = {
+            id: this.props.initialId,
```

Comment:
I don't see an `initialId` prop on this component, so this won't pass type-check. If it did, it would change the behaviour of the component which is not our aim here, so we should remove this.